modules/serverless-logic-global-config-customization.adoc

```diff
@@ -0,0 +1,120 @@
+
+[cols=3*,options="header"]
+|===
+|Configuration key
+|Default value
+|Description
+
+|`defaultPvcKanikoSize`
+|`1Gi`
+|The default size of Kaniko PVC when using the internal {ServerlessLogicOperatorName} builder manager.
+
+|`healthFailureThresholdDevMode`
+|`50`
+|How much time (in seconds) to wait for a devmode workflow to start. This information is used for the controller manager to create new devmode containers and setup the healthcheck probes.
+
+|`kanikoDefaultWarmerImageTag`
+|`gcr.io/kaniko-project/warmer:v1.9.0`
+|Default image used internally by the Operator Managed Kaniko builder to create the warmup pods.
+
+|`kanikoExecutorImageTag`
+|`gcr.io/kaniko-project/executor:v1.9.0`
+|Default image used internally by the Operator Managed Kaniko builder to create the executor pods.
+
+|`jobsServicePostgreSQLImageTag`
+|empty
+|The Jobs Service image for PostgreSQL to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`jobsServiceEphemeralImageTag`
+|empty
+|The Jobs Service image without persistence to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`dataIndexPostgreSQLImageTag`
+|empty
+|The Data Index image for PostgreSQL to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`dataIndexEphemeralImageTag`
+|empty
+|The Data Index image without persistence to use, if empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`sonataFlowBaseBuilderImageTag`
+|empty
+|{ServerlessLogicProductName} base builder image used in the internal Dockerfile to build workflow applications in preview profile. If empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`sonataFlowDevModeImageTag`
+|empty
+|The image to use to deploy {ServerlessLogicProductName} workflow images in devmode profile. If empty, the {ServerlessLogicOperatorName} will use the default Apache Community one based on the current {ServerlessLogicOperatorName} version.
+
+|`builderConfigMapName`
+|`sonataflow-operator-builder-config`
+|The default name of the builder configMap in the {ServerlessLogicOperatorName} namespace.
+
+|postgreSQLPersistenceExtensions
+|next column
+|Quarkus extensions required for workflows persistence. These extensions are used by the {ServerlessLogicOperatorName} builder in cases where the workflow being built has configured postgresql persistence. Default values are `io.quarkus:quarkus-agroal:3.8.4.redhat-00002`, `io.quarkus:quarkus-jdbc-postgresql:3.8.4.redhat-00002`, `org.kie:kie-addons-quarkus-persistence-jdbc:9.100.0.redhat-00004`
```

Comment:
Versions here are from older build, should be `9.102.redhat-0004` and `3.8.6.redhat-00004`

However as we do CR2 this will change, again consider not including the default value here.

Reply:
I will remove the default values from here.